cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.ql

```diff
@@ -3,7 +3,7 @@
  * @description Calling 'strncat' with an incorrect size argument may result in a buffer overflow.
  * @kind problem
  * @problem.severity warning
- * @security-severity 10.0
+ * @security-severity 9.3
```

Comment:
I'm surprised to see some of these drop - is that explained by the formula that converts base scores into impact scores?

Reply:
The CVE base score tends to be higher than the impact score, but not always. Impact = real-world exploitability, which is often higher in C++ due to the ability to run shell code. Looking at the list, the 75th percentile CVE has base score of 9.3 and impact score of 10.0. You can use the `explain.py` tool to dig into strange results.

I'm not concerned or surprised by this result as the category hasn't changed and feels right for the query.